src/activities/reversecount/Reversecount.qml

```diff
@@ -59,6 +59,7 @@ ActivityBase {
             property alias tux: tux
             property alias fishToReach: fishToReach
             property int clockPosition: 4
+            property int mode: 1
```

Comment:
Harcoded numbers shouldn't be used for mode: it's difficult to track to what they really correspond.
In the current case, it would be quite easy to have a difference between the configuration value and the domino value if we change one or the other and it would be complicated to spot